app/components/channel_drawer/channel_list.js

```diff
@@ -140,6 +142,10 @@ export default class ChannelList extends React.Component {
         this.props.closeChannelDrawer();
     };
 
+    handleClose = (channel) => {
```

Comment:
Sure we can discuss this more, but I have a prototype that i want to present in standup and see if we can decide on that